Pipfile

```diff
@@ -25,6 +24,7 @@ pytest = "*"
 ruff = "*"
 setuptools = "*"
 pip-audit = "*"
+boto3-stubs = {extras = ["essential"], version = "*"}
```

Comment:
Heh, alright if we arrange these in alphabetical order? 😅

Reply:
I'm game - but FWIW `uv` will enforce this eventually automatically.  I think I'm getting lazy already with `pipenv` until that magical day.